### PR TITLE
Add "Search on Forvo" action to source text context menu

### PR DESCRIPTION
**Pull request by flyvint**
i fixed your suggestions from PR #294

ps: i created new PR because github closed previous #294, when i moved my commits to branch and updated master from your repo :(

### CODE REVIEW

File: src/mainwindow.cpp [hunk 1]
```diff
@@ -381,6 +383,16 @@ void MainWindow::clearTranslation()
...
 
+void MainWindow::searchOnForvo()
+{
+    QString txt = ui->sourceEdit->textCursor().selectedText();
+    if (txt.isEmpty())
+        txt = ui->sourceEdit->toPlainText();
+    if (txt.isEmpty())
+        return;
+    QDesktopServices::openUrl(QUrl(QStringLiteral("https://forvo.com/search/%1/").arg(txt)));
+}
```
Comment (Shatur95) [inline src/mainwindow.cpp:394]: I meant to move this inside `SourceTextEdit` to connect `&QAction::triggered` directly to this method . Sorry if I was unclear :)

BTW, maybe it is worth making the same menu for translation? I think this functionality will be useful.
If you agree with my idea, I will try to move the functionality to a separate class myself to call it in both edits.

**Review by Shatur95 — CHANGES_REQUESTED** [review]
> ps: i created new PR because github closed previous #294, when i moved my commits to branch and updated master from your repo :(

No problem :)

File: src/mainwindow.cpp (hunk 1 above)
Comment (flyvint) [reply src/mainwindow.cpp:394]: :) ок

Comment (flyvint) [reply src/mainwindow.cpp:394]: about menu for translation: it would great

**Pull request merged**